### PR TITLE
Add documentation for metrics endpoint

### PR DESCRIPTION
**Pull request by calmh**

### CODE REVIEW

**Review by acolomb — COMMENTED** [review]
Looks good, some small possible corrections.

**Review by acolomb — APPROVED** [review]

File: users/metrics.rst [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Prometheus-style metrics
```
Comment (acolomb) [inline users/metrics.rst:1]:
```suggestion
Prometheus-Style Metrics
```

**Review by acolomb — CHANGES_REQUESTED** [review]
Actually, one more thing. Otherwise go ahead and merge :wink:

**Review by acolomb — APPROVED** [review]

**Comment by calmh** [comment]
👌

**Pull request merged**